<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Bt_video_on_off">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_video_off">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QWidget" name="videoContainer" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_tcp_send">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>460</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_target_user">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>380</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text_onlineUsers">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>430</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>MSG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_msg">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>420</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PTE_recv">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>280</y>
     <width>301</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
